R/check_times.R

```diff
@@ -0,0 +1,54 @@
+check_times = function(relative_paths,
+                       archive_mode = FALSE,
+                       force_backup = FALSE){
+
+  local_base = GAMBLR.helpers::check_config_value(config::get("project_base"))
+  remote_base = GAMBLR.helpers::check_config_value(config::get("project_base",config="default"))
+  if(archive_mode){
+    if(local_base == remote_base){
+      message("checking against local archive")
+    }else{
+      message("Currently, this mode must be run on the GSC (not remotely)")
+      return(NULL)
+    }
+    local_base = GAMBLR.helpers::check_config_value(config::get("archive"))
+
+  }
+  for(rel_f in relative_paths){
+    local_f = paste0(local_base,rel_f)
+    remote_f = paste0(remote_base,rel_f)
+    print(rel_f)
+    if(file.exists(local_f)){
+      mtime = file.info(local_f)$mtime
+      mtime = stringr::str_remove(mtime,"\\s\\d+:\\d+:\\d+")
+      #print(mtime)
+
+      remote_session = check_remote_configuration(auto_connect=TRUE)
+      #print(remote_f)
+      if(remote_session){
+        output = ssh::ssh_exec_internal(ssh_session,paste("stat -L ",remote_f,"| grep Modify"))$stdout
+
+        output = rawToChar(output) %>% stringr::str_extract(.,"\\d+-\\d+-\\d+")
+      }else{
+        output = file.info(remote_f)$mtime %>% stringr::str_remove("\\s\\d+:\\d+:\\d+")
+      }
+      remote_time = lubridate::as_date(output)
```

Comment:
`R/check_times.R` prefixes some funtions from `lubridate` package, but this package is not included in DESCRIPTION.